sample/simple-videoplayer-channel/Makefile

```diff
@@ -17,5 +19,4 @@ build:
 	(bsc)
 
 unit-tests:
-	(curl -d '' http://192.168.1.144:8060/launch/dev?RunTests=true)
-	# (curl -d '' 'http://192.168.1.144:8060/launch/dev?RunTests=true')
\ No newline at end of file
+	(curl -d '' http://192.168.1.144:8060/launch/dev?RunTests=true)
```

Comment:
let get ip as parameter to make command

```
make unit-test roku-server=192.168.1.144
```

Reply:
I may just remove it because we're moving to another way to launch tests.